### PR TITLE
Postgresql privs reformat

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
- fixed doc formatting (type, examples, etc)
- added return value
- pg_utils
- added aliases

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

File: lib/ansible/modules/database/postgresql/postgresql_privs.py [hunk 1]
```diff
@@ -997,7 +1046,7 @@ def main():
         conn.rollback()
     else:
         conn.commit()
-    module.exit_json(changed=changed)
+    module.exit_json(changed=changed, queries=executed_queries)
```
Comment (gundalow) [inline lib/ansible/modules/database/postgresql/postgresql_privs.py:1045]: Nice addition, thanks

File: lib/ansible/modules/database/postgresql/postgresql_privs.py [hunk 2]
```diff
@@ -826,41 +872,44 @@ def build_absent(self):
 
 
 def main():
+    argument_spec = postgres_common_argument_spec()
```
Comment (gundalow) [inline lib/ansible/modules/database/postgresql/postgresql_privs.py:875]: I think options that exist in https://github.com/ansible/ansible/blob/fbaa6c33caab85a0f71c39142d0ea7ed71e30515/lib/ansible/module_utils/postgres.py#L53-L60 should be removed from this modules argspec.

File: test/sanity/validate-modules/ignore.txt [hunk 3]
```diff
@@ -337,7 +337,6 @@ lib/ansible/modules/database/postgresql/postgresql_db.py E210
 lib/ansible/modules/database/postgresql/postgresql_ext.py E322
 lib/ansible/modules/database/postgresql/postgresql_ext.py E324
 lib/ansible/modules/database/postgresql/postgresql_lang.py E324
-lib/ansible/modules/database/postgresql/postgresql_privs.py E322
```
Comment (gundalow) [inline test/sanity/validate-modules/ignore.txt:340]: Yay!

**Review by gundalow — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/database/postgresql/postgresql_privs.py (hunk 2 above)
Comment (Andersson007) [reply lib/ansible/modules/database/postgresql/postgresql_privs.py:875]: Thank you for this hint, I removed
```
     port=dict(type='int', default=5432), 
     ssl_mode=dict(default='prefer', choices=['disable', 'allow', 'prefer', 'require', 'verify-ca', 'verify-full']), 
     ca_cert=dict(aliases=['ssl_rootcert'])
```

Comment (Andersson007) [reply lib/ansible/modules/database/postgresql/postgresql_privs.py:875]: this module seems very old, it contains several params with non-standard names.
I wouldn't like to touch them in this PR

**Review by gundalow — APPROVED** [review]

**Pull request merged**